tools/licenses/lib/main.dart

```diff
@@ -1664,7 +1664,10 @@ class _RepositoryPkgDirectory extends _RepositoryDirectory {
 
   @override
   bool shouldRecurse(fs.IoNode entry) {
-    return entry.name != 'archive';  // contains nothing that ends up in the binary executable
+    return entry.name != 'archive'  // contains nothing that ends up in the binary executable
+      && entry.name != 'file'
+      && entry.name != 'platform'
+      && entry.name != 'process';
```

Comment:
I wonder if this is safe - it seems like we might want to use `entry.fullName.endsWith` or something to avoid skipping over some random subdirectory of another thing we care about that happens to have a directory named `file` or `platform` or `process`.

Reply:
Oh I see, this class is only used for the Dart third party packages. Ok.

Reply:
Right. This only covers exactly `third_party/pkg/process` etc.